ipld/amt/src/lib.rs

```diff
@@ -5,7 +5,7 @@
 //! AMT crate for use as rust IPLD data structure
 //!
 //! Data structure reference:
-//! https://github.com/ipld/specs/blob/51fab05b4fe4930d3d851d50cc1e5f1a02092deb/data-structures/vector.md
+//! <https://github.com/ipld/specs/blob/51fab05b4fe4930d3d851d50cc1e5f1a02092deb/data-structures/vector.md>
```

Comment:
```
warning: this URL is not a hyperlink
 --> ipld/amt/src/lib.rs:8:5
  |
8 | //! https://github.com/ipld/specs/blob/51fab05b4fe4930d3d851d50cc1e5f1a02092deb/data-structures/vector.md
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: bare URLs are not automatically turned into clickable links
  = note: `#[warn(rustdoc::bare_urls)]` on by default
help: use an automatic link instead
  |
8 | //! <https://github.com/ipld/specs/blob/51fab05b4fe4930d3d851d50cc1e5f1a02092deb/data-structures/vector.md>
  |     +                                                                                                     +

```

Reply:
There are other doc links issues in the repo; I'll have a look later and see if there can be a friendly CI check to ensure this doesn't happen in the future.